sqlvm-alwayson-cluster/README.md

```diff
@@ -1,4 +1,4 @@
-# Create an Always On Availability Group with SQL Server 2014 replica virtual machines
+# Solution name
```

Comment:
No ,i will fix this. This is caused by a conflict between my change and Ninar's change.